### ISSUE TITLE
Web components & React: Tabs extended - When focus reaches to tab with text wrap, tool tip is not appearing.

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#6233]
## Detailed description

Issue:
When focus reaches to tab with text wrap, tool tip is not appearing.

Expected Result:
Tooltip should be accessible for keyboard user

**Comment by Praveen-sr** [comment]
Re tested this on web component & react, now Tooltip is accessible for keyboard user. - Working as expected.
cc @RobertaJHahn

**Comment by RobertaJHahn** [comment]
Closing based on Proveen's comments above.
